carddrawer/src/main/java/com/meli/android/carddrawer/model/CardDrawerViewMedium.java

```diff
@@ -81,4 +85,30 @@ private int getArrowColor(@NonNull @FontType final String fontType, @ColorInt fi
                 return fontColor;
         }
     }
+
+    @Override
+    protected void showGenericText(@NonNull final GenericPaymentMethod genericPaymentMethod) {
+        if (genericPaymentMethod.getDescription() != null) {
+            showText(genericPaymentMethod.getDescription());
+        } else {
+            showText(genericPaymentMethod.getSubtitle());
+        }
+    }
+
+    private void showText(@Nullable final GenericPaymentMethod.Text text) {
```

Comment:
Maybe we can reuse this for [my other comment](https://github.com/mercadolibre/meli-card-drawer-android/pull/94/files#r772606563)